documentation/references/supported-browsers.md

```diff
@@ -0,0 +1,16 @@
+---
+layout: docs
+title: Supported Browsers
+permalink: /documentation/references/supported-browsers.html
+---
+# Supported Browsers
+
+TestCafe Studio supports the following browsers:
+
+* Google Chrome: Stable, Beta, Dev and Canary
+* Internet Explorer (11+)
+* Microsoft Edge
+* Mozilla Firefox
+* Safari
+* Android browser
+* Safari mobile
```

Comment:
In the changes above we specify in which browsers we support running only, and in which we support recording.
I think this should be mentioned here as well.

What do you think?

Reply:
Yep, good not, I think we should say that we support recording only for desktop browsers.